packages/app/src/device/modules/device.ts

```diff
@@ -169,7 +159,7 @@ export class Device {
     this.strategy.off(eventName, listener)
   }
 
-  private mountDeviceListeners(): void {
+  private mountListeners(): void {
     // AUTO DISABLED - fix me if you like :)
     // eslint-disable-next-line @typescript-eslint/unbound-method
     this.on(DeviceServiceEvent.DeviceConnected, this.emitConnectionEvent)
```

Comment:
I don't think there should be still lint error with that unbound-method, you've changed function to arrow functions, so it should be fine :)